docs/de/docs/index.md

```diff
@@ -167,9 +162,9 @@ def read_item(item_id: int, q: Optional[str] = None):
 ```
 
 <details markdown="1">
-<summary>Or use <code>async def</code>...</summary>
+<summary>Oder verwende <code>async def</code>...</summary>
```

Comment:
If we stick to the formal 'Sie', it should be used here as well:
`Oder verwenden Sie`